docs/azure/packages.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Package index for Azure SDK for .NET
+description: Listing of the package names, NuGet links, docs links, and source code links for all libraries in the Azure SDK for .NET.
+ms.service: multiple
+ms.custom: azure-sdk-dotnet
```

Comment:
There is a build warning here.

> Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.

Reply:
`ms.prod` is defined in [docfx.json](https://github.com/dotnet/docs/blob/e20efe7eb096f27337081bfb9a2ef2304eac59f8/docfx.json#L168).